.ignore

```diff
@@ -0,0 +1,2 @@
+postapi/bin
+postapi/obj
```

Comment:
The filename should be .gitignore and put those bin and obj folder on this file, so that it should not be committed.

Reply:
Updated the Changes